encoding/protojson/decode_test.go

```diff
@@ -2197,6 +2197,18 @@ func TestUnmarshal(t *testing.T) {
 		wantMessage: &anypb.Any{
 			TypeUrl: "type.googleapis.com/google.protobuf.Empty",
 		},
+	}, {
+		desc:         "Any with Empty and no value and AllowPartial",
+		umo:          protojson.UnmarshalOptions{AllowPartial: true},
+		inputMessage: &anypb.Any{},
+		inputText: `{
+  "@type": "type.googleapis.com/google.protobuf.Empty"
+}`,
+		wantMessage: func() proto.Message {
+			return &anypb.Any{
+				TypeUrl: "type.googleapis.com/google.protobuf.Empty",
+			}
+		}(),
```

Comment:
This function wrapping is unnecessary. Since the function doesn’t do anything other than return an `anypb.Any` composite literal.